install.md

```diff
@@ -1,5 +1,27 @@
 # libpod Installation Instructions
 
+## Installing packaged versions of Podman
+
+#### Fedora, Centos, OpenSUSE, SUSE:
+
+yum -y install podman
+
+#### RHEL7
+
+Enable the extras repo.
+yum -y install podman
+
+#### RHEL8
+
+yum -y install podman
```

Comment:
My playbooks have it as:
```
# dnf module enable -y container-tools:1.0
# dnf module install -y container-tools:1.0
```
...but that's a month or two old, and I'm not sure if it's still current best practice, let alone what the situation might be in another month.

Reply:
Well adding it.